api/main.py

```diff
@@ -106,6 +106,13 @@ async def login_for_access_token(
                 detail=f"Invalid scope: {scope}"
             )
 
+    if 'admin' in form_data.scopes:
```

Comment:
Maybe yet another thing that could be handled by `api.auth`.

Reply:
I think this should be added to a method in the User model.

Reply:
OK let's leave it like this for now.